### PR TITLE
Make region label value conigurable

### PR DESCRIPTION
**Pull request by StanislavStefanov**
*Description**

Changes proposed in this pull request:
- make region label value configurable

### CODE REVIEW

**Review by dzahariev — APPROVED** [review]

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -127,7 +127,7 @@ global:
       version: "PR-56"
     e2e_tests:
       dir:
-      version: "PR-2263"
+      version: "PR-2283"
```
Comment (la4ezar) [inline chart/compass/values.yaml:130]: Shouldn't this be 2284?

**Pull request merged**